File: BAEKJOON/cpp/9550_아이들은_사탕을_좋아해.cpp
#include <iostream>
#include <vector>
using namespace std;

int n, k, answer = 0;
vector<int> candy;

void initialize() {
	answer = 0;
	candy.clear();
}

void input() {
	cin >> n >> k;

	int a;
	for (int i = 0; i < n; i++) {
		cin >> a;
		candy.push_back(a);
	}
}

void solution() {
	for (int i = 0; i < candy.size(); i++) {
		answer += (candy[i] / k);
	}

	cout << answer << "\n";
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	int tc;
	cin >> tc;

	for (int i = 0; i < tc; i++) {
		initialize();
		input();
		solution();
	}

	return 0;
}

File: BAEKJOON/java/9550_아이들은_사탕을_좋아해.java
import java.io.*;
import java.util.*;

public class Main {
    public static int n, k, answer = 0;
    public static List<Integer> candy = new ArrayList<>();

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    public static StringTokenizer st;

    public static void main(String[] args) throws IOException {
        int tc = Integer.parseInt(br.readLine());

        for (int i = 0; i < tc; i++) {
            initialize();
            input();
            solution();
        }

        bw.flush();
        bw.close();
    }

    private static void initialize() throws IOException {
        answer = 0;
        candy.clear();
    }

    private static void input() throws IOException {
        st = new StringTokenizer(br.readLine());

        n = Integer.parseInt(st.nextToken());
        k = Integer.parseInt(st.nextToken());

        st = new StringTokenizer(br.readLine());

        for (int i = 0; i < n; i++) {
            int a = Integer.parseInt(st.nextToken());
            candy.add(a);
        }
    }

    private static void solution() throws IOException {
        for (int i = 0; i < candy.size(); i++) {
            answer += (candy.get(i) / k);
        }

        bw.write(answer + "\n");
    }
}